constructor-manager-api/setup.cfg

```diff
@@ -23,7 +23,7 @@ classifiers =
     Topic :: Scientific/Engineering :: Image Processing
 project_urls =
     Bug Tracker = https://github.com/napari/packaging/issues
-    Source Code = https://github.com/napari/packaging/constructor-manager
+    Source Code = https://github.com/napari/packaging/constructor-manager-api
```

Comment:
Should `pyyaml` be added in the `install_requires` section of this package? I tried doing:

```python
from constructor_manager_api.api import check_updates
```

and I got an error related to the `import yaml` done for the settings module:

```python-traceback
>>> from constructor_manager_api.api import check_updates
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "E:\Acer\Documentos\Quansight\Napari\packaging-otros\gonzalo\packaging\constructor-manager-api\src\constructor_manager_api\api.py", line 10, in <module>
    from constructor_manager_api.utils.settings import save_settings
  File "E:\Acer\Documentos\Quansight\Napari\packaging-otros\gonzalo\packaging\constructor-manager-api\src\constructor_manager_api\utils\settings.py", line 5, in <module>
    import yaml
ModuleNotFoundError: No module named 'yaml'
```

Reply:
Yes you are right!